recipes-support/intel-aero-repo/intel-aero-repo/intel-aero.repo

```diff
@@ -0,0 +1,4 @@
+[Intel Aero]
+name=Intel Aero
+baseurl=https://download.01.org/aero/repo/1.4.0/
```

Comment:
I think this should be 1.4, not 1.4.0... so we can create updates during a single release.